scripts/dispatch-torque.py

```diff
@@ -120,7 +122,7 @@ def get_output(command):
     handle = sys.argv[2]
     response = {
         'handle': handle,
-        'raw': get_output('qdel %s' % handle)
+        'raw': get_output(ssh(['/usr/bin/qdel', handle]))
```

Comment:
Thanks - switched to /usr/bin/env

On Sun, Nov 15, 2015 at 8:21 PM, Stephen Koo <notifications@github.com>
wrote:

> In scripts/dispatch-torque.py
> <https://github.com/codalab/codalab-cli/pull/229#discussion_r44888083>:
>
> > @@ -120,7 +122,7 @@ def get_output(command):
> >      handle = sys.argv[2]
> >      response = {
> >          'handle': handle,
> > -        'raw': get_output('qdel %s' % handle)
> > +        'raw': get_output(ssh(['/usr/bin/qdel', handle]))
>
> same as above
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/229/files#r44888083>.
>